src/components/RepoList/RepoList.js

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import CardView from '../CardView/';
+
+const RepoList = ({ isLoading, repos }) => {
+  if (isLoading) {
+    return <p>Loading...</p>;
+  }
+
+  if (repos && repos.length) {
+    return <div className="card-columns">
+      {repos.map(repo =>
+        <CardView
+          key={repo.id}
+          title={repo.name}
+          text={repo.description}
+          imageUrl={repo.avatar}
+          imageAlt={`Owner: ${repo.author}`}
+          imageWidth={180}
+          imageHeight={180}
+          buttonUrl={repo.url}
+          buttonText="View on Github"
+        />
+      )}
+    </div>;
+  } else {
+    return <p>No repositories to display.</p>
```

Comment:
yes, you're right, the condition should be `if (repos && repos.length === 0) { return <p>No repositories to display.</p>; }`